### PR TITLE
Update POLIS related entries

### PR DESCRIPTION
**Pull request by hlooman**
POLIS was disabled when Cryptopia went out of business. This PR is adding all new coin related changes.
Ratefetcher updates and coin DIP3 updates. See commits for details.

### CODE REVIEW

**Comment by hlooman** [comment]
Note: one of the circle.ci integrationtests failed, can you check if I added the PolisPay exchange at the right spot as it only responds for POLIS_BTC requests. It is fully functional in all other currencies with standard rate fetching via CoinGecko though.

Re: /BTCPayServer.Rating/Services/RateProviderFactory.cs line 82

File: BTCPayServer.Common/Altcoins/BTCPayNetworkProvider.cs [hunk 1]
```diff
@@ -77,7 +77,7 @@ public BTCPayNetworkProvider(NetworkType networkType)
             }
 
             // Disabled because of https://twitter.com/Cryptopia_NZ/status/1085084168852291586
-            //InitPolis();
+            InitPolis();
```
Comment (Kukks) [inline BTCPayServer.Common/Altcoins/BTCPayNetworkProvider.cs]: Move this next to the other enabled ones

Comment (hlooman) [reply BTCPayServer.Common/Altcoins/BTCPayNetworkProvider.cs]: Done - made suggested changes to rate provider as well, it is now passing integration tests.

**Comment by hlooman** [comment]
It's now not completing the selenium tests while nothing really changed... It was passing before. It has issues `Cannot assign requested address` which looks like something in the testing system rather than in the code.

**Comment by Kukks** [comment]
Let me re run the tests. It was failing on an unrelated issue( we test all links on each page to see if any of them are dead. One of them was not responding when you ran the tests)

**Comment by Kukks** [comment]
@pavlenex  Did we do some changes to https://docs.btcpayserver.org/features/wallet 
The page was not responding for a while( it is up for me now though)

**Comment by hlooman** [comment]
Ok, thank you.

**Comment by pavlenex** [comment]
@Kukks I've made quite a few changes to that page [7 days ago.](https://github.com/btcpayserver/btcpayserver-doc/commit/05de48588a0e97cc136822017ccae6e5bba6ea47)

**Pull request merged**